sdcm/cluster_k8s/__init__.py

```diff
@@ -1960,6 +1961,9 @@ def refresh_ip_address(self):
 
         self._init_port_mapping()
 
+    @retrying(n=60, sleep_time=10,
+              allowed_exceptions=(k8s_exceptions.ApiException, invoke.exceptions.UnexpectedExit),
```

Comment:
is kubectl runs only with `LocalRunner()` ? if it uses ssh2 based remoter, it has it's own UnexpectedExit that need to be added as well

Reply:
We use only ` LocalRunner()` for kubectl commands.